### ISSUE TITLE
Damaged backup versions

### ISSUE THREAD

**Issue by AmorimJ** [LukeMurphey/lookup-editor#2]
Hi @LukeMurphey,
There's a bug in the backups history.
The thing is, so you save a lookup (do a backup), not a big deal, but then, if you remove a column and save again the lookup, the previous backups of the lookup will be damaged because the save will override the previous versions without the column that you just deleted.
Best regards,
João Amorim

**Comment by LukeMurphey** [comment]
I'm not seeing this behavior. Here is what I tried to repro the behavior:

1. Made a lookup with 5 columns
2. Saved the lookup several times; confirmed that backups load
3. Removed a column
4. Saved the lookup
5. Attempted to load a backup that had less columns (it worked)

**Comment by AmorimJ** [comment]
Ok Luke,

 

But if you have a backup lookup saved with 5 columns and then you remove one and save it again, it is supposed that the backup won't change the number of columns or is it?

 

Best regards

 

De: Luke [mailto:notifications@github.com] 
Enviada: 25 de novembro de 2015 23:35
Para: LukeMurphey/lookup-editor
Cc: AmorimJ
Assunto: Re: [lookup-editor] Damaged backup versions (#2)

 

I'm not seeing this behavior. Here is what I tried to repro the behavior:

1.	Made a lookup with 5 columns
2.	Saved the lookup several times; confirmed that backups load
3.	Removed a column
4.	Saved the lookup
5.	Attempted to load a backup that had less columns (it worked)
